psakefile.ps1

```diff
@@ -0,0 +1,48 @@
+include "./psake-build-helpers.ps1"
+
+properties {
+	$configuration = 'Release'
+	$projectRoot = Resolve-Path "./"
+	$buildDir = "$projectRoot/Build"
+	$testResultsDir = "$projectRoot/src/BulkWriter.Tests/TestResults"
+	$nugetVersion = "3.0.0"
+	$version = "3.0.0-ci-local"
+}
+
+task default -depends Run-Tests
+task Run-CI -depends Clean-Solution, Run-Tests -description "Continuous Integration process"
+task Run-LocalCI -depends Clean-Solution, Run-Tests, Package-Local -description "Local continuous integration process"
+task ReBuild-Solution -depends Clean-Solution, Run-Tests -description "Rebuild the code, with testing"
+
+task Get-Info -description "Display runtime information" {
+	exec { dotnet --info }
+}
+
+task Run-Tests -depends Run-Restore, Build-Solution -description "Run unit tests" {
+	exec { dotnet test --configuration $configuration  --logger "trx;LogFileName=Testresults.xml" } -workingDirectory src/BulkWriter.Tests
+}
+
+task Build-Solution -depends Get-Info -description "Build the solution" {
+	if($version -like '*tags-v*') {
```

Comment:
I still don't agree with this `$nugetVersion` approach. If you apply a tag of `releases/3.0.0` then the generated version should be `3.0.0`. Otherwise, it will generate a version based on your branch name and building from master will calculate a version that makes sense (and you can include a suffix for master as well).